gpii/node_modules/processReporter/src/ProcessReporter.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * GPII Process Reporter
+ *
+ * Copyright 2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("kettle", require);
+fluid.require("./processesBridge.js", require);
+
+fluid.defaults("gpii.processReporter", {
+    gradeNames: "fluid.component"
+});
+
+fluid.defaults("gpii.processReporter.base", {
+    gradeNames: ["kettle.app", "gpii.processReporter"],
+    requestHandlers: {
+        processGet: {
+            route: "/processes",
+            type: "gpii.processReporter.allSolutions.get",
+            method: "get"
+        },
+        processGetSingle: {
+            route: "/processes/:solutionID",
+            type: "gpii.processReporter.solution.get",
+            method: "get"
+        }
+    },
```

Comment:
@kaspermarkus @amb26 I'm finally getting to this business of the RESTful implementation of the process reporter.

I want to confirm that we are abandoning REST in this case -- is that so?  If so, what do you want to replace it with?  Two possibilities come to mind:  another component that the local FlowManager has a reference to, and uses in the same node process, or the "electron-IPC channel" that Antranig suggests.  Which do you prefer?